src/meta/model_v2/src/table.rs

```diff
@@ -118,6 +118,7 @@ pub struct Model {
     pub value_indices: I32Array,
     pub definition: String,
     pub handle_pk_conflict_behavior: HandleConflictBehavior,
+    pub version_column_index: Option<u32>,
```

Comment:
also need to change the https://github.com/risingwavelabs/risingwave/blob/8972479890a22b798751ce739e6eaa8e2e39fb56/src/meta/model_v2/migration/src/m20230908_072257_init.rs#L1079

Reply:
@yezizp2012 To be honest I am not sure what we need to do on the migration script, could you help to give some advice? 🥵